### PR TITLE
added comments to new server.js and npm init

### PR DESCRIPTION
**Pull request by TheMichaelRay**
I worked so hard on this

@philuchansky

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Initial upload
```
Comment (philuchansky) [inline server.js]: Hey @TheMichaelRay can you please change "upload" to "commit". Thanks lolz.

**Comment by philuchansky** [comment]
@TheMichaelRay check the commentary, please update asap and I'll merge lolz.

**Comment by TheMichaelRay** [comment]
Hey @philuchansky I just updated the code. lolz.

**Comment by philuchansky** [comment]
@TheMichaelRay looks good. I'll merge this and we can celebrate with champagne.

**Comment by philuchansky** [comment]
lolz

**Pull request merged**